src/docker_snapshot/container.py

```diff
@@ -28,59 +41,69 @@ def wrapper(*args, **kwargs):
 
 
 class freeze_target_container(object):
-    def __init__(self):
+    def __init__(self) -> None:
         pass
 
-    def __enter__(self):
+    def __enter__(self) -> None:
         stop(settings.get("container_name"))
 
-    def __exit__(self, type, value, traceback):
+    def __exit__(
+        self,
+        exception_type: t.Optional[t.Type[BaseException]],
+        exception: t.Optional[BaseException],
+        traceback: t.Optional["TracebackType"],
+    ) -> None:
         start(settings.get("container_name"))
```

Comment:
Small note: this could technically also be simplified to a `contextlib.contextmanager`, but I can fix that later.

Reply:
Hm, yeah probably...